### ISSUE TITLE
Feature Request: Add Support for Neo4j Graph Database

### ISSUE THREAD

**Issue by tuddman** [metabase/metabase#2711]
[neo4j](http://www.neo4j.com)

**Comment by AdamGhani** [comment]
any updates?

**Comment by camsaul** [comment]
Hi @AdamGhani, we're adding support for various DBs based on community demand. Since only a few people have asked for Neo4j thus far, it might be a while before the core team gets around to it. 

If you're familiar with Clojure and feeling adventurous you can try [writing a driver yourself](https://github.com/metabase/metabase/wiki/Writing-A-Driver). A few of our drivers (Presto, Vertica and Crate) were contributed by community members. I'm available to help with any questions you might run into.

Anyone else interested in this driver, please add a :+1: reaction to the issue description rather than adding `+1` or 👍  comments. One way we prioritize issues is by the number of 👍  reactions.

**Comment by Lucaszw** [comment]
Any updated plans on this?

**Comment by Vadorequest** [comment]
Neo4j driver could be achieved through GraphQL Driver
GraphQL driver would allow to connect Metabase to a lot of other system, through GraphQL itself. 

https://github.com/metabase/metabase/issues/6297

**Comment by Pushplaybang** [comment]
neo4j has a fantastic query interface through cypher,  and adding graphQL into the mix seems like it wouldn't be that beneficial for the analytics work, given the discussion you've linked to, and the various difficulties such as lack of standardisation.

**Comment by Vadorequest** [comment]
Indeed, my suggestion to use GraphQL is older and probably isn't such a great idea. See https://github.com/metabase/metabase/issues/6297#issuecomment-496010940

**Comment by weslien** [comment]
This is more relevant than ever, graph databases (and Neo4j in particular) are seeing wide adoption, and the lack of support for any graph database is the only thing stopping me from doing a PoC of metabase for my company. The combination would be interesting indeed!

**Comment by bbenzikry** [comment]
I have a little pet project that uses the Neo4j BI connector for SQL and allows for Cypher queries in the editor. 
I didn't spend a lot of time on it due to prior obligations ( so be warned :smile: ), but it seems to work well for most of our use cases ( mostly centralities / path algorithm results ) 

https://github.com/bbenzikry/metabase-neo4j-driver
